app/src/main/java/org/wikipedia/main/MainActivity.kt

```diff
@@ -0,0 +1,189 @@
+package org.wikipedia.main
+
+import android.content.Context
+import android.content.Intent
+import android.os.Bundle
+import android.view.Menu
+import android.view.View
+import androidx.appcompat.view.ActionMode
+import androidx.appcompat.widget.Toolbar
+import androidx.fragment.app.Fragment
+import org.wikipedia.Constants
+import org.wikipedia.R
+import org.wikipedia.WikipediaApp
+import org.wikipedia.activity.SingleFragmentActivity
+import org.wikipedia.appshortcuts.AppShortcuts.Companion.setShortcuts
+import org.wikipedia.databinding.ActivityMainBinding
+import org.wikipedia.navtab.NavTab
+import org.wikipedia.onboarding.InitialOnboardingActivity
+import org.wikipedia.page.PageActivity
+import org.wikipedia.page.tabs.TabActivity
+import org.wikipedia.settings.Prefs
+import org.wikipedia.suggestededits.SuggestedEditsTasksFragment
+import org.wikipedia.util.DimenUtil.dpToPx
+import org.wikipedia.util.DimenUtil.getDimension
+import org.wikipedia.util.FeedbackUtil.setButtonLongPressToast
+import org.wikipedia.util.FeedbackUtil.showTooltip
+import org.wikipedia.util.ResourceUtil.getThemedColor
```

Comment:
Should we either use `DimenUtil.dpToPx()` or directly use `dpToPx()` kind of methods? I usually use the first one, which makes more sense to me.

Reply:
Yes, this is my preference too.  For some reason the automatic Kotlin converter prefers to import individual methods, which are hard to catch and clean up.  (It does this for `L.e()`, transforming it to `e()` which is super annoying)